Cards effects cannot be split up (#18)
GiottoVerducci/rulebook2024: content.md

> another card. Even drawing to replace cards, comes after this time
> period.

> Cards effects resolve in full: they are not interruptible nor pre-emptible, and their effects cannot be "split up" in a sequence unless they are in an explicit sequence.

Reviewer: ```suggestion
Any effect can only be used if all of its requirements are met. Likewise, the resolution of a single effect always resolves in full. They are not interruptible nor pre-emptible, and their resolution cannot be split unless they are in an explicit sequence. 
```

I tried to clarify a bit, also that you need to have an equipment / retainer in hand if Disguised Weapon / Pack Alpha is used to bring them into play. With this change you shouldn't be able to DI the equipment, only "Disguised Weapon" as well.

The weapon must be announced and in hand when the card is played, before it is replaced, but need not to be shown: it can be kept apart face down. 

[[LSJ 19991207]](https://groups.google.com/g/rec.games.trading-cards.jyhad/c/aQaOTYwC-fg/m/vdyo09QZoicJ) 
[[ANK 20180925-2]](https://www.vekn.net/forum/rules-questions/77029-order-of-draw-and-replace-for-concealed-weapon-under-infernal-pursuit?start=6#90757) 
[[LSJ 20100130]](https://groups.google.com/g/rec.games.trading-cards.jyhad/c/X8Uu7Sk56P4/m/twUvR4ZnBBkJ)

> Any effect can only be used if all of its requirements are met. The resolution of a single effect always resolves in full. They are not interruptible nor pre-emptible, and their resolution cannot be split unless they are in an explicit sequence. 

Author: ```suggestion
Effects can only be used if their requirements are met. They always resolve in full. They are not interruptible nor pre-emptible, and their resolution cannot be split unless they set up an explicit sequence. 
```

> For example, "Discard X cards" requires you to discard all X cards at once: you cannot discard them one by one and replace in between.
> On the other hand, "Combat ends, and inflict 1 damage to the opposing minion" sets up an explicit sequence: if another card played afterward prevents the combat from ending, the rest of the sequence is not executed (the vampire does not unlock).

Author: ```suggestion
For example, "Discard X cards" requires you to discard all X cards at once: you cannot discard them one by one and replace in between.
"Combat ends, and inflict 1 damage to the opposing minion" sets up an explicit sequence: if a card played afterward prevents the combat from ending, the rest of the sequence is not executed (damage is not inflicted).
```
